elfi/client.py

```diff
@@ -164,21 +167,18 @@ def num_cores(self):
 class ClientBase:
     """Client api for serving multiple simultaneous inferences"""
 
-    # TODO: add the self.tasks dict available
-    # TODO: test that client is emptied from tasks as they are received
-
     def apply(self, kallable, *args, **kwargs):
```

Comment:
Maybe `kallable` should be documented as this is a Base class. The naming might not be immediately obvious.